### PR TITLE
Addressing issues

### PR DESCRIPTION
**Pull request by j-clap**
* Corrected port formatting
* Now reading app default credentials
* Removed Time-zone-specific flaky test
* Fixed merge conflicts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ except for some cases concerning PSQL, wherein the query itself is translated.
 
 Simple query mode and extended query mode are supported, and any data type
 supported by Spanner is also supported. Items, tables and language not native to
```
Comment (bvandiver) [inline README.md]: plural, plural, singular I think is the grammar issue here.

File: README.md [hunk 2]
```diff
@@ -14,15 +14,15 @@ except for some cases concerning PSQL, wherein the query itself is translated.
...
+Spanner is not supported, unless otherwise specified.
 
 Though the majority of functionality inherent in most PostgreSQL clients
 (including PSQL and JDBC) are included out of the box, the following items are
 not supported:
-* Functions
-* COPY protocol
-* Prepared Statement DESCRIBE
-* SSL
+* Functions.
```
Comment (bvandiver) [inline README.md]: Nit: bulleted lists of nouns don't often have '.'s

File: src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java [hunk 3]
```diff
@@ -119,6 +120,28 @@ private int buildProxyPort(CommandLine commandLine) {
...
 
+  /**
+   * Get credential file path from either command line or application default. If neither throw
+   * error.
+   *
+   * @param commandLine The parsed options for CLI
+   * @return The absolute path of the credentials file.
+   */
+  private String buildCredentialsFile(CommandLine commandLine) {
+    System.out.println("here");
```
Comment (bvandiver) [inline src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java]: Remove?

File: src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java [hunk 4]
```diff
@@ -119,6 +120,28 @@ private int buildProxyPort(CommandLine commandLine) {
...
+    if (!commandLine.hasOption(OPTION_CREDENTIALS_FILE)) {
+      String credentialsPath = Credentials.getApplicationDefaultCredentialsFilePath();
+      if (credentialsPath == null) {
+        throw new IllegalArgumentException(
+            "User must specify a valid credential file, "
+            + "or have application default credentials set-up.");
+      }
+      System.out.println(credentialsPath);
```
Comment (bvandiver) [inline src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java]: ditto

**Review by bvandiver — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (j-clap) [reply README.md]: Sorry accidentally placed an old version of README. Replaced with current.

File: README.md (hunk 2 above)
Comment (j-clap) [reply README.md]: idem

File: src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java (hunk 3 above)
Comment (j-clap) [reply src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java]: Oops.Done.

File: src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java (hunk 4 above)
Comment (j-clap) [reply src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java]: Done.

**Pull request merged**